integration/test/roster_test.js

```diff
@@ -0,0 +1,312 @@
+const { describe, before, after, it } = require('mocha');
+const { v4: uuidv4 } = require('uuid');
+const RosterTestPage = require('../pages/RosterTestPage');
+const DriverFactory = require('../utils/DriverFactory');
+
+describe('Roster Test', async function () {
+  const chromeDriverFactory = new DriverFactory();
+  const firefoxDriverFactory = new DriverFactory();
+  let localPage;
+  let remotePage;
+  const meetingId = uuidv4();
+  localAttendeeName = 'local-attendee';
+  remoteAttendeeName = 'remote-attendee';
+  let failureCount = 0;
+
+  before(async function () {
+    await chromeDriverFactory.build('Roster Test - Local', 'chrome');
+    localPage = new RosterTestPage(chromeDriverFactory.driver);
+
+    await firefoxDriverFactory.build('Roster Test - Remote', 'firefox');
+    remotePage = new RosterTestPage(firefoxDriverFactory.driver);
+  });
+
+  afterEach(function () {
+    const state = this.currentTest.state;
+    if (state === "failed") {
```

Comment:
nit: Use single quotes for consistency.

Reply:
Checked the double quotes again as a whole and fixed them. Should be find now.